### PR TITLE
how-to/maintain: update slow log document (new)

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Refine some words in slow log document
- Refine some SQL and shell scripts in the document

### What is the related PR or file link(s)? 
pingcap/docs-cn#1752
#1462
https://github.com/pingcap/docs/pull/1489 (PR with unknown error)

### Which version does your change affect? 
dev, v2.1, v3.0

### CODE REVIEW

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 1]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
 ```
 
 Due to the memory footprint restriction, the stored slow query records count is limited. If the specified `N` is greater than the records count, the returned records count may be smaller than `N`.
+
+The table below specifies the details of the output:
+
+| Column name | Description |
+|:------|:---- |
+| start | The starting time of SQL execution |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| start | The starting time of the SQL execution |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 2]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| duration | The duration of SQL execution |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| duration | The duration of the SQL execution |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 3]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| details | The details of SQL execution |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| details | The details of the SQL execution |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 4]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| succ | Whether a SQL statement is executed successfully. `1` means success and `0` means failure. |
+| conn_id | The Connection ID (session ID). For example, you can use the keyword `con:3` to query the log whose session ID is `3`. |
+| transcation_ts | The `commit ts` that a transaction commits |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| transcation_ts | The `commit ts` for a transaction commit |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 5]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| user | The username used for executing the statement |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| user | The user name for the execution of the SQL statement |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 6]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| db | The database involved for executing the statement |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| db | The database involved when the SQL statement is executed |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 7]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| table_ids | The table ID involved for executing a SQL statement |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| table_ids | The ID of the table involved when the SQL statement is executed |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 8]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| index_ids | The index ID involved for executing a SQL statement |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| index_ids | The ID of the index involved when the SQL statement is executed |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 9]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| internal | The SQL statement inside TiDB |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| internal | This is a TiDB internal SQL statement |
```

File: v2.1/how-to/maintain/identify-slow-queries.md [hunk 10]
```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
...
+| digest | The fingerprint of a SQL statement |
```
Comment (TomShawn) [inline v2.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
| digest | The fingerprint of the SQL statement |
```

File: v2.1/how-to/maintain/identify-slow-queries.md (hunk 1 above)
Comment (lilin90) [reply v2.1/how-to/maintain/identify-slow-queries.md]: @TomShawn `the` here is not a must. Let's just keep one `the` in this phrase to make it sound simple.

Comment (TomShawn) [reply v2.1/how-to/maintain/identify-slow-queries.md]: OK.

**Comment by lilin90** [comment]
@juliezhang1112 Please address comments from @TomShawn for all three versions.

**Comment by juliezhang1112** [comment]
PTAL @lilin90 @TomShawn

File: dev/how-to/maintain/identify-slow-queries.md [hunk 11]
```diff
@@ -59,111 +72,128 @@ select count(1) from t_slim, t_wide where t_slim.c0>t_wide.c0 and t_slim.c1>t_wi
...
+
+    ```sql
+    select query_time, query, digest
+    from information_schema.slow_query
+    where is_internal = false
+    order by query_time desc
+    limit 1;
+    ```
+
+    Usage example
```
Comment (lilin90) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
    Usage example:
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 12]
```diff
@@ -59,111 +72,128 @@ select count(1) from t_slim, t_wide where t_slim.c0>t_wide.c0 and t_slim.c1>t_wi
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select query, query_time
+    from information_schema.slow_query
+    where digest = "4751cb6008fda383e22dacb601fde85425dc8f8cf669338d55d944bafb46a6fa";
+    ```
+
+    Usage example
```
Comment (lilin90) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
    Usage example:
```

**Pull request merged**